### ISSUE TITLE
Compartment search doesn't yield correct total

### ISSUE THREAD

**Issue by orlandow** [Aidbox/Issues#44]
## Data:

Patient: pat1
Conditions: c1 and c2 (referencing pat1)

## Queries

These two queries should be equivalent but they yield different totals:

**With compartments**
```
GET /Patient/pat1/Condition?_count=1
--> total: 1
```

**Without compartments**
```
GET /Condition?subject=pat1&_count=1
--> total: 2
```

**Comment by orlandow** [comment]
Even if we don't specify the `_count` the total always reflects the payload total that is returned by the server

**Comment by mlapshin** [comment]
fixed in `0.3.4-3-g763f3de`
